src/ctap/data_formats.rs

```diff
@@ -481,7 +525,8 @@ impl From<PublicKeyCredentialSource> for cbor::Value {
             RpId => Some(credential.rp_id),
             UserHandle => Some(credential.user_handle),
             OtherUi => credential.other_ui,
-            CredRandom => credential.cred_random
+            CredRandom => credential.cred_random,
+            CredProtectPolicy => credential.cred_protect_policy.map(|p| p as i64),
```

Comment:
Is there a reason not to implement `From<CredentialProtectionPolicy> for cbor::Value`? We could remove the `map(|p| p as i64)` in that case.

Reply:
The other enum that is a glorified i64 also didn't, I changed both.